object_follower_node:
  ros__parameters:
    linear_speed: 0.2      # линейная скорость робота
    angular_speed: 0.5     # угловая скорость робота
    stop_area: 30000.0     # площадь объекта, при которой нужно остановиться
    hsv_lower1: [0, 120, 70]       # нижняя граница HSV для первого диапазона красного
    hsv_upper1: [10, 255, 255]     # верхняя граница HSV для первого диапазона красного
    hsv_lower2: [170, 120, 70]     # нижняя граница HSV для второго диапазона красного
    hsv_upper2: [180, 255, 255]    # верхняя граница HSV для второго диапазона красного